fastexcel-writer/src/main/java/org/dhatim/fastexcel/Range.java

```diff
@@ -44,6 +44,11 @@ public class Range implements Ref {
      */
     private final int right;
 
+    /**
+     * Check about the Range visibility when a named range is added.
```

Comment:
I find this comment confusing. Doesn't seem to turn on any "Check", but rather control the scope of this range (right?)